mlibrary/lib.umich.edu
# February package updates

# Overview
Monthly dependency update to help maintain lib.umich.edu.

[https://deploy-preview-278--future-wwwlib-previews.netlify.app](https://deploy-preview-278--future-wwwlib-previews.netlify.app)

## NPM
Check the `package.json` file to see all dependency updates. 

## Testing
- Install the updated packages (`npm install`).
- Start [the site](http://localhost:8000) and check if everything still works (`npm start`).
- Make sure the PR is consistent in these browsers:

This update excludes the breaking changes from `rehype-parse`, `rehype-react`, and `unified` that impact the [html component](https://github.com/mlibrary/lib.umich.edu/blob/master/src/components/html.js). 

I have [a ticket](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-103) to circle back and fix the html.component file.
